providers/themeProvider.tsx

```diff
@@ -0,0 +1,25 @@
+import { ThemeProvider as NextThemeProvider } from 'next-themes';
+import { ThemeProvider as MUIProvider } from '@mui/material/styles';
+import { createTheme } from '@mui/material/styles';
+import { MotionConfig } from 'framer-motion';
+import { useMemo, type FC, type PropsWithChildren } from 'react';
```

Comment:
wouldn't it be easier to read something like this?
```suggestion
import { useMemo } from 'react';
import type { FC, PropsWithChildren  } from 'react';
```

Reply:
I have no idea why the freak my commit made this.

Reply:
But this sounds like something new, didn't know you could import types like this nowadays hmmm...